.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,17 @@
-<!--- Provide a link to the ticket or document which prompted this change -->
-
 ##### SUMMARY
-<!--- Describe the change below, including rationale and design decisions -->
+<!--
+    Provide a link to the ticket or document which prompted this change,
+    Describe the rationale and design decisions.
+-->
 
 ##### FEATURE FLAG
-<!--- If this is specific to a feature flag, which one? -->
+<!-- If this is specific to a feature flag, which one? -->
 
 ##### RISK ASSESSMENT / QA PLAN
-<!-- Does this need QA before or after merge? Link QA ticket. -->
+<!--
+    Is there test coverage? Is there manual QA planned? Link QA ticket.
+    Is there anything a deployer should know about rolling back this change?
```

Comment:
Should we make a note somewhere about what get's included in the pre-deploy messages? I think only the title, and labels are printed: https://github.com/dimagi/commcare-cloud/blob/c4a9553edb6cf2de93a3051e7ce4b97963a5cb73/src/commcare_cloud/fab/utils.py#L337-L343

Reply:
I don't quite follow, so maybe this could be phrased better. I wasn't thinking about the person who deploys the PR, but rather someone who goes to revert the PR because it's caused a problem, figuring that person re-reads the description and should be aware if there's anything special they need to know before reverted & re-deploying.